contracts/Dispenser.sol

```diff
@@ -79,6 +190,14 @@ contract Dispenser is IErrorsTokenomics {
         }
     }
 
+    function setRemainingServiceStakingAmount(uint256 epochNumber, uint256 amount) external {
+        // Check for the tokenomics
+        if (msg.sender != tokenomics) {
+            revert ManagerOnly(msg.sender, tokenomics);
+        }
+        mapEpochRemainingServiceStakingAmounts[epochNumber] = amount;
+    }
+
```

Comment:
Add a notice to claimOwnerIncentives() function saying that it is pausable independently of Treasury pausabilty